src/_guides/bump-sh-tutorials/fastapi.md

```diff
@@ -31,3 +33,31 @@ The following assumes your local machine is configured with Python and FastAPI,
   ```
 
 That's it! Enjoy the comfort of Bump.sh to browse through your API doc, and [customize it to your needs](/help/getting-started/quick-start#customization-options).
+
+## Enriching documents
+
+By default, FastAPI generates what can be considered a plain technical description of your API. Although it is robust and complete, it's not enough for people discovering your API without any upfront knowledge about it, access to the code, or ability to test it extensively. Such documents require contextual information, and any appropriate adjustements.
+
+FastAPI lets you extend documents, [using Python](https://fastapi.tiangolo.com/how-to/extending-openapi/). 
+
+You can also accomplish this using OpenAPI's [Overlay specification](https://github.com/OAI/Overlay-Specification/blob/main/versions/1.0.0.md), which can be convenient in all use cases where you do not want to change your Python code base (e.g. when involving Technical Writers or Product Managers in writing docs).
+
+The Bump.sh CLI supports Overlays. Technically, the `bump overlay` command will output a modified version of the `[DEFINITION_FILE]` (an OpenAPI or AsyncAPI document) by applying the operations described in the `[OVERLAY_FILE]` Overlay file to the original API document.
+
+1. Create your Overlay document (see example use cases in [Augmenting Generated OpenAPI Documents with Filters & Overlays](/guides/openapi/augmenting-generated-openapi))
+
+2. Apply your Overlay document to the original API document with the Bump.sh CLI
+```shell
+bump overlay openapi.json overlays.json > openapi.updated.json
+```
+
+3. Deploy the resulting file to Bump.sh
+  ```shell
+  bump deploy openapi.updated.json \
+    --doc my-documentation-name \
+    --token my-documentation-token
+  ```
```

Comment:
Real question: should we push for the 2 step process of `bump overlay` + `bump deploy` or should we mostly suggest to use the combined version of `bump deploy --overlay` in a single step?

Reply:
Good point, I've updated the guide!